src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -5246,41 +5102,548 @@ namespace HVACVariableRefrigerantFlow {
             }
         } // IF(ALL(TerminalUnitList(VRFTU(VRFTUNum)%TUListIndex)%IsSimulated))THEN
 
-        // one-time check to see if VRF TU's are on Zone Equipment List or issue warning
+        // one-time check to see if VRF TU's are on ZoneHVAC:EquipmentList or AirloopHVAC or issue warning
```

Comment:
Added checks here to find data for zone/airloop/OA system TUs. This is currently done only once for all TUs. This code is prime for rework to only check the TU that is called which will help with having enough data to simulate without unnecessary IF checks throughout code.